app/lib/ui/views/new/new_screen.dart

```diff
@@ -17,17 +17,146 @@ class NewScreen extends StatelessWidget {
+                            color: Colors.blueGrey,
+                            fontWeight: FontWeight.bold,
+                            fontSize: 10.0),
+                      )),
+                  Positioned(
+                      bottom: 0.0,
+                      right: 0.0,
+                      child: IconButton(
+                        icon: Icon(Icons.edit),
+                        color: Colors.blueGrey,
+                        onPressed: () {},
+                      )),
+                  Positioned(
+                      child: CircleAvatar(
+                    radius: 50,
+                    backgroundColor: Colors.white,
+                    backgroundImage: NetworkImage(
+                        "https://images.unsplash.com/photo-1544393569-eb1568319eef?ixid=MXwxMjA3fDB8MHxzZWFyY2h8MXx8c3RvcHxlbnwwfHwwfA%3D%3D&ixlib=rb-1.2.1&w=1000&q=80"),
+                  ))
+                ],
               ),
+              ListView(
+                shrinkWrap: true,
+                padding: const EdgeInsets.all(10.0),
+                children: <Widget>[
+                  RaisedButton.icon(
+                    onPressed: model.startNewAtr,
+                    padding: EdgeInsets.all(30.0),
+                    icon: Icon(
+                      Icons.add_circle,
+                      color: Colors.white,
+                      size: 40.0,
+                    ),
+                    color: Colors.orange,
+                    label: Text(
+                      'New Form',
+                      style: TextStyle(
+                          fontStyle: FontStyle.italic,
+                          fontSize: 20.0,
+                          fontWeight: FontWeight.bold,
+                          color: Colors.white),
+                      textAlign: TextAlign.left,
+                    ),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(
+                      top: 10.0,
+                    ),
+                  ),
+                  RaisedButton.icon(
+                    onPressed: model.startNewAtr,
+                    padding: EdgeInsets.all(30.0),
+                    icon: Icon(
+                      Icons.wifi_protected_setup,
+                      color: Colors.white,
+                      size: 40.0,
+                    ),
+                    color: Colors.orange,
+                    label: Text(
+                      'Active Form',
+                      style: TextStyle(
+                          fontStyle: FontStyle.italic,
+                          fontSize: 20.0,
+                          fontWeight: FontWeight.bold,
+                          color: Colors.white),
+                    ),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(
+                      top: 10.0,
+                    ),
+                  ),
+                  RaisedButton.icon(
+                    onPressed: model.startNewAtr,
```

Comment:
Lmao, all the buttons start a new ATR huh.

Reply:
@holtzmak yea, because I don't know which model should I use for active and history. Did you use `model.HistoryScreenViewModel` for history and `mode.saveEditedAtr` or `mode.saveCompletedAtr` for active screen?

Reply:
I have the fix for this in my latest. I'll handle it